src/qt/rpcconsole.cpp

```diff
@@ -471,14 +471,28 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     constexpr QChar nonbreaking_hyphen(8209);
     const std::vector<QString> CONNECTION_TYPE_DOC{
+        //: Explanatory text for an inbound peer connection.
         tr("Inbound: initiated by peer"),
+        /*: Explanatory text for an outbound peer connection which
+            relays all network information. This is the default behavior for
+            Outbound connections. */
         tr("Outbound Full Relay: default"),
+        /*: Explanatory text for an outbound peer connection which relays
```

Comment:
throughout: either precede "which" with a comma and space, or use "that", e.g.

"connection, which relays"

"connection that relays"

Reply:
thanks, updated